aloha-core/src/test/scala/com/eharmony/aloha/semantics/compiled/CompiledSemanticsTest.scala

```diff
@@ -108,6 +143,8 @@ class CompiledSemanticsTest {
     }
 }
 
+object CompiledSemanticsTest extends LazyInitSupport
```

Comment:
Added the `code` function to the companion object so that it can be used in the spec in the test to determine if `code` is called.